RELEASING.md

```diff
@@ -7,16 +7,13 @@ Run linting:
 Set variables:
 
     $ export VERSION=X.Y.Z
-    $ export GPG_KEY=EA456E8BAF0109429583EED83578F667F2F3A5FA
+    $ export GPG_KEY=20EE002D778AE197EF7D0D2CB993FF98A90C9AB1
```

Comment:
@niklasmohrin if you have a GPG signing key, you can replace this with your own key ID after the release.